src/serlio/scripts/serlioCreateUI.mel

```diff
@@ -51,10 +51,27 @@ global proc createPrtNode() {
 	string $rulePackage[] = `fileDialog2 -fm 1 -cap "Select Rule Package" -ff $filters`;
 	
 	
+	int $alreadyHasExistingPrtNode = false;
+
 	if(size($rulePackage)) {
 		for($node in $initialShapes) {
 			select -r $node;
-			serlioAssign $rulePackage;
+			string $nodeHistory[] = `listHistory $node`;
+
+			for($dpNode in $nodeHistory){
+				$dpNodeType = `nodeType $dpNode`;
+				if($dpNodeType == "serlio"){
+					setAttr -type "string" ($dpNode + ".Rule_Package") $rulePackage[0];  
+					evalDeferred "refreshEditorTemplates()";
+					$alreadyHasExistingPrtNode = true;
+				}
+			}
+
+			if (!alreadyHasExistingPrtNode){
```

Comment:
Missing `$` character, I think.

// Error: "F:/dev/git/serlio/install/scripts/serlioCreateUI.mel" line 70.34: Invalid use of Maya object "alreadyHasExistingPrtNode". //